### ISSUE TITLE
User can see the right answer before replying - can we avoid it?

### ISSUE THREAD

**Issue by TiagoJacobs** [h5p/h5p-flashcards#68]
Hello, thanks for this amazing solution.

However I have one concern using it. If user inspect the network tab, it can see the `content.json` file, and this way see the expected answer for each question.

Is there any way to avoid this?

![image](https://user-images.githubusercontent.com/2075267/106693625-9d735780-65b5-11eb-84ae-081bc232ac58.png)

**Comment by tdjac0bs** [comment]
Just to be clear, an exploit that any student could use would be something like this:
```
JSON.parse(window.H5PIntegration.contents["cid-vv6nhmwki"].jsonContent).cards.map(card => card.answer)
```

So it will get all the answers:
![image](https://user-images.githubusercontent.com/2075267/106697912-8127e880-65be-11eb-8e82-5cba6c7262c2.png)

**Comment by otacke** [comment]
@tdjac0bs You are correct. There will always be ways to get hold of the solutions when using software that evaluates the answers in the browser.
The H5P core team has server-side evaluation on the roadmap for H5P however, see https://h5p.org/roadmap

**Comment by otacke** [comment]
@TiagoJacobs I'll close this ticket as answered. Feel free to create a new one if you have follow-up questions.
